src/item.cpp

```diff
@@ -2,6 +2,7 @@
 #include "player.h"
 #include "output.h"
 #include "skill.h"
+#include "monster.h"
```

Comment:
Sorry about that, legacy from when I was trying to get the monster item encapsulation working with a pointer to the monster class. It isn't necessary, so I'll get rid of it.